### PR TITLE
typo - fixed quotes

### PR DESCRIPTION
**Pull request by macfarla**
Signed-off-by: Sally MacFarlane <sally.macfarlane@consensys.net>

typo

- [x] Doc content
- [ ] Doc pages organisation

### CODE REVIEW

File: docs/HowTo/Configure/Consensus-Protocols/Add-Validators.md [hunk 1]
```diff
@@ -468,7 +468,7 @@ To bypass the smart contract and specify new validators:
                "blockperiodseconds": 2,
                "epochlength": 30000,
                "requesttimeoutseconds": 4,
-               “validatorcontractaddress”: “0x0000000000000000000000000000000000007777”
+               "validatorcontractaddress”: “0x0000000000000000000000000000000000007777"
```
Comment (alexandratran) [inline docs/HowTo/Configure/Consensus-Protocols/Add-Validators.md:471]:
```suggestion
               "validatorcontractaddress": "0x0000000000000000000000000000000000007777"
```

Comment (macfarla) [reply docs/HowTo/Configure/Consensus-Protocols/Add-Validators.md]: fixed that one and a few more. Is there a way to find and replace these special characters?

Comment (alexandratran) [reply docs/HowTo/Configure/Consensus-Protocols/Add-Validators.md]: Yes IntelliJ distinguishes between the two characters. Find and replace using shift + cmd + R.

**Review by alexandratran — APPROVED** [review]

**Comment by NicolasMassart** [comment]
Can you please fix the link in CONTRIBUTING.md at the same time as it fails your checks?
change to `https://wiki.hyperledger.org/display/BESU/Reporting+bugs+in+the+documentation`

**Pull request merged**